builtin/providers/azurerm/resource_arm_managed_image.go

```diff
@@ -0,0 +1,421 @@
+package azurerm
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/arm/compute"
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceArmManagedImage() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmManagedImageCreate,
+		Read:   resourceArmManagedImageRead,
+		Update: resourceArmManagedImageCreate,
+		Delete: resourceArmManagedImageDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": locationSchema(),
```

Comment:
Right now it's Name, Resource Group Name, and OS Disk that have ForceNew applied - can you think of any other properties that should have that?